meeting-notes/20181113.md

```diff
@@ -0,0 +1,203 @@
+   - peer-star-app:
+     - fix: DiasSet break points: https://github.com/ipfs-shipyard/peer-star-app/pull/101
+     - fix: deduplicates membeship addresses: https://github.com/ipfs-shipyard/peer-star-app/pull/102
+     - pinner should only get state: https://github.com/ipfs-shipyard/peer-star-app/pull/106
+     - test pinner: https://github.com/ipfs-shipyard/peer-star-app/pull/108
+     - membership heuristic sampling function should be synchronous: https://github.com/ipfs-shipyard/peer-star-app/pull/110
+     - chore: using aegir lint and fixes linting errors: https://github.com/ipfs-shipyard/peer-star-app/pull/116
+ - In progress:
+    - expose collaboration replication events: https://github.com/ipfs-shipyard/peer-star-app/pull/111
+ - Blocked:
+ - Next:
+    - continue on exposing collaboration replication events: https://github.com/ipfs-shipyard/peer-star-app/pull/111
+    - support different strategies for local store: https://github.com/ipfs-shipyard/peer-star-app/issues/117
+
+#### @satazor
+ - Concluded:
+   - working group 2019 roadmap, 1st iteration
+   - discussify:
+     - UI level QA: https://github.com/ipfs-shipyard/pm-discussify/issues/58
+     - Standardized styleguide comment names with @andreforsousa
+ - In progress:
+   - Replies
+   - Lazy load of comments and replies (view more)
+ - Next:
+   - Implement "show history" in the UI
+   - Add persistance of the data (pinner + sync UI language)
+     
+Note: Ask pedro about granularity of Discussify v1 milestones on the roadmap
+     
+ #### @andreforsousa    
+ - Concluded:
+    - Discussify:
+    - Style-guide updates with all scenarios and states (hover, focus, error feedback) for "comment-component"
+        - Comment
+        - Comment (ownership)
+        - Comment Actions
+        - Comment Edited
+        - Comment Edited History
+        - Comment Placeholder
+        - Comment Removed
+        - Comment Input (new reply)
+        - Comment Input (editing)
+    - QA & style-guide updates for the pre-alpha version
+        - FAB Shadow fix
+    - PeerPad UX/UI fixes:
+        - Grid width for 80columns accommodation (desktop)
+        - Dismissible notifications (desktop)
+        - Empty state illustration
+        - History revision
+        
+- Blocked:
+    - PeerPad mobile:
+    - Hi-Fi prototype to start looking for:
+        - Transitions between elements
+        - Start defining values for the eases
+        - This will be in the backlog until desktop version is properly defined and polished, following David’s feedback.
+    
+- In progress:
+  - Discussify Brand & Web Guidelines (manual)
+  - Look at new features for PeerPad (sync with Victor)
+
+ 
+#### @VictorBjelkholm
+ - Concluded:
+ 		- Error reporting on peerpad.net
+ - In Progress:
+ 		- UX review
+    - Pinner review
+ - Blocked:
+ - Next:
+    - Pinner deployment
+    - Metrics
+ 
+#### @aschmahmann
+ - Concluded:
+  - Roadmapping round #1
+  - Basic graph sync + updating working
```

Comment:
@aschmahmann mind record a demo of it working? Where can we find the code for this?

Reply:
Right now the demo of it working is just Go tests, I'm working this week on putting together a demo chat application that utilizes this. I'll put together the repo and link it here today or tomorrow.

Reply:
For now the repo to follow is: https://github.com/aschmahmann/ipshare